src/main/java/io/appform/dropwizard/actors/base/UnmanagedConsumer.java

```diff
@@ -59,14 +63,15 @@ public UnmanagedConsumer(final String name,
         this.queueName = NamingUtils.queueName(config.getPrefix(), name);
         this.retryStrategy = retryStrategyFactory.create(config.getRetryConfig());
         this.exceptionHandler = exceptionHandlingFactory.create(config.getExceptionHandlerConfig());
+        this.observer = connection.getRootObserver();
```

Comment:
Ensure not null, otherwise this will break during runtime

Reply:
Added Observer null check in RabbitmqActorBundle.java